MMM-social-counter.js

```diff
@@ -23,6 +23,7 @@ Module.register('MMM-social-counter', {
     this.twitter = this.config.twitter;
     this.updatesEvery = this.config.updatesEvery;
     this.size = this.config.size;
+    this.icon = this.config.icon || 'fa-twitter';
```

Comment:
I'd rather have all default values in one overview at the `defaults` object at the top of this file. Can you move this please?